appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java

```diff
@@ -117,6 +117,10 @@
   @DefaultMessage("Checkpoint ...")
   @Description("Label of the button for checkpoint")
   String checkpointButton();
+  
+  @DefaultMessage("Create checkpoint before deleting"")
```

Comment:
I had a compile fail probably caused by the quotes typo here.

Reply:
Resolved the quotes typo

Reply:
It looks like there are now merge conflicts in the PR.